head

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
I'd suggest appending a `.sh` suffix, since this is a shell-script, i.e. `head.sh`. Maybe also add a verb, i.e. `run_head.sh` (if run is an appropriate prefix here).

Reply:
Basically it is deployed to /usr/bin, so there should be no extension from my point of view.

Reply:
Viewed in this light, I agree :+1: